Services/Data/default.json

```diff
@@ -63,12 +63,13 @@
   ],
   "Rules": [
     {
-      "Id": "default_Chiller_Pressure_High",
```

Comment:
Revert, we need Id field

Reply:
should we change this for telemetry service too?

Reply:
Actually the one in telemetry service should be deleted to avoid confusion as it is not being used.